tests/sources/test_github.py

```diff
@@ -1023,14 +1042,73 @@ async def test_fetch_files():
 @pytest.mark.asyncio
 async def test_get_docs():
     expected_response = [
-        REPOS,
+        PUBLIC_REPO,
         MOCK_RESPONSE_PULL,
         MOCK_RESPONSE_ISSUE,
         MOCK_RESPONSE_ATTACHMENTS[0],
     ]
     actual_response = []
     async with create_github_source() as source:
-        source._fetch_repos = Mock(return_value=AsyncIterator([REPOS]))
+        source._fetch_repos = Mock(return_value=AsyncIterator([deepcopy(PUBLIC_REPO)]))
```

Comment:
using `deepcopy` otherwise the global objects get mutated, which influences other tests